### PR TITLE
Add Ideas-2021.md

### PR DESCRIPTION
**Pull request by chimosky**
@perriefidelis @JuiP @llaske @walterbender @aperezbios kindly review, will make more changes but I decided to create the page first.

### CODE REVIEW

File: Ideas-2021.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# GSoC 2021 Ideas
+
+[Administrative notes](#administrative-notes)
+
+------------
+# Administrative notes
+
+Above are a list of ideas we've planned for GSoC 2020 projects.
```
Comment (JuiP) [inline Ideas-2021.md:8]: GSoC 2020 should be changed to GSoC 2021

**Review by JuiP — COMMENTED** [review]
Everything else looks fine! :)

**Comment by JuiP** [comment]
@chimosky 
Included 2 relevant Sugar project ideas from Ideas-2020.md with a little modification.
You may review and cherry-pick commit: 4cc1994 
I will add others soon...

**Comment by chimosky** [comment]
> @chimosky
> Included 2 relevant Sugar project ideas from Ideas-2020.md with a little modification.
> You may review and cherry-pick commit: [4cc1994](https://github.com/sugarlabs/GSoC/commit/4cc1994e26185bd193fdb84851769b1f3bee44d3)
> I will add others soon...

Seems 4cc1994 doesn't belong to this repo and I can't find a fork if it belongs to one.

**Comment by chimosky** [comment]
Also something worthy of being mentioned, this year's GSoC has changed in some ways and I'll highlight them below so anyone who's adding any idea would have it in mind;

```
Smaller project size - all students participating in the 2021 program will be working on a 175 hour project 
(instead of a 350 hr project). 
This change will also result in a few other changes including the student stipend being cut in half. 

Currently we are missing out on many wonderful students who could never commit to such a huge project
and time commitment but would be great contributors to your community. This is a significant change as we 
now are no longer strongly encouraging students to focus only on GSoC over their summer.
Students have many other responsibilities especially during the pandemic that make it hard for them to spend
30 hours a week on a project. 
We realize this is going to require all of you to think about smaller projects and update your project ideas
which is why we wanted to give you 3+ months to start talking it through with your communities.

Shortened coding period - the coding period will be 10 weeks with a lot more flexibility for the mentor and
student to decide together how they want to spread the work out over the summer.
Some folks may choose to stick to a 17-18 hour a week schedule with their students, others may factor in a
couple of breaks during the program (for student and mentor) and some may have students focus 30 hours a
week on their project so they wrap up in 6 weeks.
This also makes it a lot easier for students with finals or other commitments (weddings, etc.)
to adjust their schedules.

2 evaluations  (instead of 3) - There will be an evaluation after 5 weeks and the final evaluation will take
place after the 10th week. 
We are also no longer requiring students complete their first evaluation (though we encourage them to do so),
so if a student doesn’t complete the first evaluation they will not automatically be removed from the program.
They are still required to complete the final evaluation.

Eligibility requirements - In 2020 there are many ways students are learning and we want to acknowledge that
so we will be allowing students who are 18 years old AND currently enrolled (or accepted into) a post-secondary
academic program as of May 17, 2021 or have graduated from a post-secondary academic program between
December 1, 2020 and May 17, 2021 to apply to the GSoC program.
What this means is that now the program will be open to folks participating in a variety of different academic
programs, not just accredited university programs. This includes licensed coding camps, community colleges,
and many other programs that may not be accredited yet but are post-secondary academic programs.
 ``` 

This means we have to make adjustments to the project ideas being added.

File: Ideas-2021.md (hunk 1 above)
Comment (chimosky) [reply Ideas-2021.md]: Thanks, fixed in 821923e.

**Comment by JuiP** [comment]
> Seems 4cc1994 doesn't belong to this repo and I can't find a fork if it belongs to one.

I do not have the rights to make changes to this repository directly. Here is the fork: https://github.com/JuiP/GSoC/tree/pr104.

Also keeping in mind the reduction in time I changed 25 activities to 15, I hope that is a good number.

**Comment by quozl** [comment]
> @chimosky I do not have the rights to make changes to this repository directly.

Added @JuiP and @llaske to repository access list.

**Comment by quozl** [comment]
https://github.com/sugarlabs/GSoC/commit/4cc1994e26185bd193fdb84851769b1f3bee44d3 reviewed.  I suggest making it into a a pull request.

**Pull request merged**